src/main/java/frc/robot/Constants.java

```diff
@@ -75,12 +75,76 @@ public final class Constants {
     // bumpers
     public static final int XBOX_LB = 5;
     public static final int XBOX_RB = 6;
-    
+
     public static final int XBOX_BACK = 7;
     public static final int XBOX_START = 8;
 
     // joy stick button
     public static final int XBOX_JL = 9;
     public static final int XBOX_JR = 10;
 
+    // TODO: The following constants came from the 2022 robot.
```

Comment:
Jack, please reference our discussion 1-2 weeks ago: The constants for a specific Subsystem (apart from CAN IDs) belong in the Subsystem class, not in Constants.

Reply:
I will merge out from main once your new branch that reorganizes constants is merged to main

Reply:
Nevermind, I will move it